tests/aws/services/ses/test_ses.py

```diff
@@ -918,6 +918,79 @@ def test_special_tags_send_email(self, tag_name, tag_value, aws_client):
 
         assert exc.match("MessageRejected")
 
+    # we cannot really introspect received emails in AWS
+    @markers.aws.only_localstack
+    def test_ses_sns_topic_integration_send_email_ses_destination(
+        self,
+        sns_topic,
+        sns_subscription,
+        ses_configuration_set,
+        ses_configuration_set_sns_event_destination,
+        setup_email_addresses,
+        aws_client,
+    ):
+        """
+        Validates that configure Event Destinations and sending an email does not trigger an infinite loop of sending
+        SNS notifications that sends an email that would trigger SNS.
+        """
+
+        sender_email_address, recipient_email_address = setup_email_addresses()
+        config_set_name = f"config-set-{short_uid()}"
+
+        EMAILS.clear()
```

Comment:
I would ask to use the REST endpoint for SES email introspection because tests are not guaranteed to run in the same process as LocalStack.

- GET `/_aws/ses`
- DELETE `/_aws/ses`

Reply:
Sure, makes sense 👍 will update

Reply:
Pinging @giograno: Are there test fixtures for the [LocalStack SDK](https://docs.localstack.cloud/aws/tooling/localstack-sdks/python-sdk/#sqs)?